pkg/authorization/registry/selfsubjectrulesreview/storage.go

```diff
@@ -22,12 +28,48 @@ func (r *REST) New() runtime.Object {
 
 // Create registers a given new ResourceAccessReview instance to r.registry.
 func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, error) {
-	// the input object has no valuable input, so don't bother checking it.false
+	rulesReview, ok := obj.(*authorizationapi.SelfSubjectRulesReview)
+	if !ok {
+		return nil, kapierrors.NewBadRequest(fmt.Sprintf("not a SelfSubjectRulesReview: %#v", obj))
+	}
+	namespace := kapi.NamespaceValue(ctx)
+	if len(namespace) == 0 {
+		return nil, kapierrors.NewBadRequest(fmt.Sprintf("namespace is required on this type: %v", namespace))
+	}
+	user, exists := kapi.UserFrom(ctx)
+	if !exists {
+		return nil, fmt.Errorf("user missing from context")
+	}
+
 	policyRules, err := r.ruleResolver.GetEffectivePolicyRules(ctx)
```

Comment:
> previously, would this let you check your cluster-level access as well?

No, it was a namespace scoped resource, so there was no way to call it like that.